test/MUXControlsTestApp/VisualTreeTestHelper.cs

```diff
@@ -0,0 +1,318 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Common;
+using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+using System.IO;
+using MUXControls.TestAppUtils;
+using MUXControlsTestApp.Utilities;
+using System.Text;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Markup;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+using static MUXControls.TestAppUtils.VisualTreeDumper;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using PlatformConfiguration = Common.PlatformConfiguration;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    public enum Theme
+    {
+        Dark,
+        Light,
+        All,
+        None
+    }
+    class VisualTreeLog
+    {
+        public static void LogInfo(string info)
+        {
+            Log.Comment(info);
+        }
+        public static void LogDebugInfo(string debugInfo) {}
+    }
+
+    public class VisualTreeTestHelper
+    {
+        public static bool AlwaysLogMasterFile = false;
```

Comment:
```suggestion
// Log MasterFile to MusicLibrary or LocalFolder. By default(AlwaysLogMasterFile=false), It logs the master files to  LocalFolder. 
// You can change the setting by set AlwaysLogMasterFile = true. After you run the tests and test case fails, the master files are put into MusicLibrary.
// Finally you can copy master files to master/ directory and check in with code. 

        public static bool AlwaysLogMasterFile = false;
```

Reply:
Sounds like based on @ranjeshj comment there's precedent for using Photos in WUX (Not sure why not Documents?). It'd be nice to match where we put these kinds of things.

You say "not mess up [your] dev machine". Are you putting them into a subfolder to keep it from getting too obnoxious?

Reply:
Document folder needs more settings, and I followed this doc [' You must add File Type Associations to your app manifest that declare specific file types that your app can access in this location. '](https://docs.microsoft.com/en-us/windows/uwp/files/file-access-permissions) but I run into permission problem. Our test project is organized different from simple C# project, and I gave up after 20 minutes.
For me, Photos and Musics are the same.
I didn't create subfolders because I run into some permission problem too.
We can fix it in the future if we do think it's a big problem, but in this PR, just keep Music and no subfolder.